internal/jimmhttp/rebac_admin/identities.go

```diff
@@ -120,17 +120,24 @@ func (s *identitiesService) GetIdentityGroups(ctx context.Context, identityId st
 		Relation:     ofganames.MemberRelation.String(),
 		TargetObject: openfga.GroupType.String(),
 	}, int32(filter.Limit()), filter.Token()) // #nosec G115 accept integer conversion
-
 	if err != nil {
 		return nil, err
 	}
+
 	groups := make([]resources.Group, len(tuples))
 	for i, t := range tuples {
+		dbGroup, err := s.jimm.GetGroupByUUID(ctx, user, t.Target.ID)
```

Comment:
It could but I didn't want to optimise prematurely. Curious to hear other opinions. If it becomes an issue we can fix it later? Makes me wonder if we have monitoring on our Rebac-admin endpoints.

Reply:
We have, i've added the middleware monitoring in this pr: https://github.com/canonical/jimm/pull/1412